src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Project name: Duke
+ * @author Tan Jun Da A023489eU
+ */
+package seedu.duke;
+
+import java.util.Scanner;
+
+/**
+ * Represents the main program of Duke.
+ */
+public class Duke {
+
+    private Ui ui;
+    private TaskList tasks;
+    private Storage storage;
+
+    /**
+     * Represents the set of commands by the user.
+     */
+    public enum Commands {
+        bye,
+        mark,
+        unmark,
+        list,
+        todo,
+        deadline,
+        event,
+        delete,
+        find
+    }
+
+    /**
+     * Constructor for the Duke class.
+     *
+     * @param filepath The file path for the data stored in the txt file.
```

Comment:
According to the Checkstyle that we are using, I think there is no extra line between "Constructor ..." and "@param ...". Please do check on this.

Reply:
I checked the coding standard textbook. There is a spacing between the method description and the @params. May i know where you find it on the Checkstyle that says no extra line?

Reply:
My bad, you are right. I have been following the warnings produced by the IntelliJ Checkstyle using the checkstyle.xml.